connman/include/rtnl.h

```diff
@@ -47,17 +49,25 @@ void connman_rtnl_remove_watch(unsigned int id);
 struct connman_rtnl {
 	const char *name;
 	int priority;
+	bool handle_rtprot_ra;
```

Comment:
If this is a public API exposed to external plugins then sticking new fields into the middle of the structure doesn't look like a good idea. On the other hand, connman doesn't seem to care much about stability of the plugin interface, this change doesn't make much of a difference, feel free to ignore.

Reply:
And there's also global variable called `handle_rtprot_ra` in `src/rtnl.c` - what's the relationship between those?

Reply:
In normal use, without the plugin those IPv6 `RTPROT_RA` messages should not be processed. The `rtnl.c` option indicates simply that those kind of messages are required to be processed, usually the RA added content is left for kernel to handle but with these blacklisted interfaces we need to see also those changes. 

I'm not really comfortable with the idea of breaking the API either but each plugin registering for IPv6 route changes may not want to handle the RA generated netlink messages. This is really a special case when the blacklisted interfaces cause issues. Other way would be that the message type is included in the callback function, which would not break the API. I'll see if that is cleaner.

Reply:
Dealt with as little as possible damage to the API. Marking resolved.